src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -466,7 +475,7 @@ export class SearchSource {
     }
   }
 
-  private flatten() {
+  flatten() {
```

Comment:
didn't see other places call this method, do we need to change the access modifier here?

Reply:
used in test